File: Examples/Java/SDK/src/main/java/com/aspose/imaging/cloud/examples/images/UpdateTiffPropertiesWithoutStorage.java
package com.aspose.imaging.cloud.examples.images;

import com.aspose.imaging.cloud.examples.Utils;
import java.io.IOException;
import java.nio.file.Files;
import java.nio.file.Path;
import java.nio.file.StandardCopyOption;

public class UpdateTiffPropertiesWithoutStorage {

    public static void main(String... args) throws IOException {
        String input = "Sample1.tiff";
        String output = "Sample2.tiff";
        Path inputFile = Utils.getPath(UpdateTiffPropertiesWithoutStorage.class, input);
        Path outputFile = Utils.getPath(UpdateTiffPropertiesWithoutStorage.class, output);

        String compression = "ccittfax3";
        String resolutionUnit = "inch";
        Integer bitDepth = 1;
        Boolean fromScratch = false;
        Float horizontalResolution = 96.0f;
        Float verticalResolution = 96.0f;
        String outPath ="";

        com.aspose.imaging.model.ResponseMessage apiResponse = Utils.getImagingSdk().PostProcessTiff (
                compression, 
                resolutionUnit, 
                bitDepth, 
                fromScratch, 
                horizontalResolution, 
                verticalResolution, 
                outPath, 
                inputFile.toFile()
        );

        Files.copy(apiResponse.getInputStream(), outputFile, StandardCopyOption.REPLACE_EXISTING);

    }
}

File: Examples/Node.js/REST/images/UpdateTiffPropertiesWithoutStorage.js
var utils = require("../utils.js");
var fs = require('fs');

var input_file = 'sample1.tiff';
var inputPath = getPath(__filename, input_file);
var output_file = 'output.tiff';
var outputPath = getPath(__filename, output_file);

var input_buffer = fs.readFileSync(inputPath);

var method = 'POST';
var request_url ='imaging/tiff?compression=ccittfax3&bitDepth=1';

ProcessCommandStream(
    method,
    Sign(request_url),
    input_buffer,
    function (output_buffer) {
        fs.writeFileSync(outputPath, output_buffer);
    }
);